scripts/remote_deployment/.ipc/evm_keystore.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "address": "0xf161ba5e71ea64b34e95a860f3b093425c7ca9af",
```

Comment:
My understanding was that we'll be creating new wallets per deployment. That would make every deployment more isolated and not dependent on any changes to these wallets. Are we using these not to use Calib faucet? (not sure what was the answer to whether the faucet exposes an API)

Reply:
Maybe these could be checked into CI as secrets? I mean, sure, their values aren't too sensitive, but if the private keys are here, a huckster could deplete them from funds just to stop the tests from passing.

Reply:
@maciejwitowski Turns out I don't have permission to add secrets for GIthub Actions in this repo. Can you grant me permission for that?